extensions/wikia/CreateNewWiki/tasks/ImportStarterData.php

```diff
@@ -21,8 +21,15 @@ public function check() {
 	}
 
 	public function run() {
-		// I need to pass $this->sDbStarter to CreateWikiLocalJob::changeStarterContributions
-		$starterDatabase = Starters::getStarterByLanguage( $this->taskContext->getLanguage() );
+		$language = $this->taskContext->getLanguage();
+
+		// TODO Clean up in XW-2393
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)